src/openms/include/OpenMS/DATASTRUCTURES/ToolDescription.h

```diff
@@ -36,6 +36,7 @@
 
 #include <OpenMS/DATASTRUCTURES/Param.h>
 #include <OpenMS/DATASTRUCTURES/String.h>
+#include <OpenMS/DATASTRUCTURES/ListUtils.h>
```

Comment:
are these includes of ListUtils.h always needed? I saw them in quite some files

Reply:
The include often is only needed for the typedef of StringList, it could be removed by adding the typedef to each file or just writing vector<String> instead of StringList in the header files